specs-go/config.go

```diff
@@ -529,7 +539,28 @@ type LinuxSeccompArg struct {
 
 // LinuxSyscall is used to match a syscall in Seccomp
 type LinuxSyscall struct {
-	Name   string             `json:"name"`
-	Action LinuxSeccompAction `json:"action"`
-	Args   []LinuxSeccompArg  `json:"args,omitempty"`
+	Name     string            `json:"name,omitempty"`
+	Names    []string          `json:"names,omitempty"`
+	Action   Action            `json:"action"`
+	Args     []LinuxSyscallArg `json:"args"`
+	Comment  string            `json:"comment"`
+	Includes Filter            `json:"includes"`
```

Comment:
@mrunalp there are no such things in libseccomp to support these conditionals (includes, excludes), it has to be implemented by the runtime author.  Instead of having every runtime handle this, the runtime author should just not add the rule if they don't want it based on setting a cap.